.changeset/chatty-actors-stare.md

```diff
@@ -0,0 +1,33 @@
+---
+'astro': minor
+---
+
+Experimental redirects support
+
+This change adds support for the redirects RFC, currently in stage 3: https://github.com/withastro/roadmap/pull/587
+
+Now you can specify redirects in your Astro config:
+
+```js
+import { defineConfig } from 'astro/config';
+
+export defineConfig({
+  redirects: {
+    '/blog/old-post': '/blog/new-post'
+  }
+});
+```
+
+You can also specify spread routes using the same syntax as in file-based routing:
+
+```js
+import { defineConfig } from 'astro/config';
+
+export defineConfig({
+  redirects: {
+    '/blog/[...slug]': '/articles/[...slog]'
```

Comment:
I’m guessing this is a typo and not a Freudian slip for what a slog it was putting this together 😅 
```suggestion
    '/blog/[...slug]': '/articles/[...slug]'
```